filebeat/module/nginx/access/machine_learning/datafeed_response_code.json

```diff
@@ -0,0 +1,44 @@
+{
+    "job_id": "JOB_ID",
```

Comment:
To clarify, I don't mean a textual replacement. The json gets decoded and the key replaced.